draft-ietf-detnet-dp-sol-mpls-02.xml

```diff
@@ -149,22 +147,33 @@
     service and forwarding sub-layer functions and supports flow identification as described in the
     DetNet Architecture.
 
-   As part of the service sub-layer functions, this document describes the
-   and how a DetNet Relay/Edge/Transit nodes works. It also describes the
+   As part of the service sub-layer functions, this document describes
+   DetNet node data plane operation. It also describes the
    function and operation of the Packet Replication (PRF) Packet Elimination
    (PEF) and Packet Ordering (POF) functions with an MPLS data plane.
 
     It also describes an MPLS-based DetNet forwarding sub-layer that eliminates (or
     reduces) contention loss  and provides bounded latency for DetNet flows.
   </t>
   <t>
-   This document does not define, but does allow for, associated controller plane
-   and Operations, Administration, and Maintenance (OAM) functions.
+    MPLS encapsulated DetNet flows can be carried over
+    network technologies that can provide the DetNet required level
+    of service.  This document defines examples of such, specifically
+    carrying MPLS DetNet flows over IEEE 802.1 TSN sub-networks, and
+    over DetNet IP PSN.
+  </t>
+  <t>
+    The intent is for this document to support different traffic types
+    being mapped over DetNet MPLS, but this is out side the scope of this
+    document. An example of such can be found in <xref

```

Comment:
What do we ntend to say here? Does it refer to L2/L3 App-flow? Or somewhat else? What is covered in the IP data-plane draft?